app/build.gradle.kts

```diff
@@ -1,98 +1,105 @@
 plugins {
-	id("com.android.application")
-	id("kotlin-android")
-	id("kotlin-android-extensions")
+    id("com.android.application")
+    id("kotlin-android")
+    id("kotlin-android-extensions")
```

Comment:
Why all the indentation changes here?

Reply:
Android studio did that automatically because it seems that it wasn't formatted correctly